tests/sessionsSpec.js

```diff
@@ -0,0 +1,68 @@
+describe("Sessions", function(){ //describe your object type
+    
+    //declare controller
+  var $controller,
+    scope,
+    $rootScope,
+    $sessionStorage,
+    ApiJsonFactory,
+    $mdDialog;
+
+
+  beforeEach(module("oe.sessions")); //load module
+
+  beforeEach(inject(function($injector, _ApiJsonFactory_) { //inject the controller
+    $controller = $injector.get("$controller");
+    $mdDialog = $injector.get("$mdDialog");
+    $rootScope = $injector.get("$rootScope");
+    $sessionStorage = $injector.get("$sessionStorage");
+    ApiJsonFactory = _ApiJsonFactory_;
+    scope = $rootScope.$new();
+    ctrlLoader = function () {
+      $controller("SessionsController", {
+        $scope: scope,
+        $mdDialog: $mdDialog
+      })};
+    });
+  }));
```

Comment:
Missing semicolon.<br>Expected an identifier and instead saw ')'.<br>Expected an assignment or function call and instead saw an expression.